### ISSUE TITLE
Coordinate differences between vectors and wide vectors

### ISSUE THREAD

**Issue by TimSylvester-mousebird** [mousebird-consulting-inc/WhirlyGlobe#1476]
Coordinates are handled slightly differently leading to gaps in small objects when drawing both with the same inputs.  Setting `kMaplyVecCentered: false` for vectors seems to improve this somewhat, implying that wide vectors should support that same functionality.

![image](https://user-images.githubusercontent.com/71895881/145658999-123a55da-8046-4829-807b-5be74a1a36b0.png)

**Comment by 0x73** [comment]
any changes?

**Comment by TimSylvester-mousebird** [comment]
@0x73 No, nothing has been done to address this.  It only occurs at very small scales, and isn't a priority for the client in whose project I noticed it.

**Comment by 0x73** [comment]
@TimSylvester-mousebird  I got it. I have same issue. Could you check it? Do you have any suggestion for me? Thanks.
<img width="979" alt="Screen Shot 2022-01-10 at 21 45 49" src="https://user-images.githubusercontent.com/2377685/148821851-84d52baa-04c1-4200-b9c2-ced479d667ef.png">

**Comment by TimSylvester-mousebird** [comment]
The problem referred to in this issue is about coordinate precision and centering, and I would guess is unrelated to issues with such large items, though I can't rule it out.

Specific to the lines not following the edges of the areas, my first inclination would be toward different subdivide options.  The subdividing process will need to make the same changes to both sets of points, or one needs to be created from the other after it's already already-subdivided, in order for the results to match up.
